appveyor.yml

```diff
@@ -1,29 +1,112 @@
 environment:
   NewPascalVersion: v1.0.50
 
-install:
-  - appveyor AddMessage -Category Information "Downloading NewPascal %NewPascalVersion%..."
-  - appveyor DownloadFile https://github.com/newpascal/newpascal/releases/download/np-%NewPascalVersion%/newpascal.zip
-  - appveyor AddMessage "Unpacking NewPascal %NewPascalVersion%..."
-  - 7z x -y "newpascal.zip" -o"C:\" > nul
-  - appveyor AddMessage -Category Information "Configuring NewPascal..."
-  - c:\newpascal\configure.bat
-  - set FPCDIR=c:\newpascal\fpcsrc
-  - set PATH=c:\newpascal\fpc\bin\i386-win32;c:\newpascal\binw32;C:\Program Files (x86)\NSIS;%path%
-
-build_script:
-  - appveyor AddMessage -Category Information "Building UltraStar-Deluxe..."
-  - c:\newpascal\lazarus\lazbuild src\ultrastardx-win.lpi --lazarusdir=c:\newpascal\lazarus
-  - appveyor AddMessage -Category Information "Done."
-
-after_build:
-  - appveyor AddMessage -Category Information "Creating Installer..."
-  - xcopy game\*.dll installer\dependencies\dll /y
-  - makensis "installer\UltraStar Deluxe.nsi"
-  - appveyor AddMessage -Category Information "Done."
-
-artifacts:
-  - path: game
-    name: UltraStar.Deluxe_continuous_portable
-    type: zip
-  - path: installer\dist\UltraStar.Deluxe_*_installer.exe
+image:
+  - Visual Studio 2015
+  - Ubuntu2004
+  - Ubuntu
+
+configuration:
+  - x86-32
+  - x86-64
+
+matrix:
+  exclude:
+    - image: Visual Studio 2015
+      configuration: x86-64
+
+for:
+  # Windows
+  -
+    matrix:
+      only:
+        - image: Visual Studio 2015
+
+    install:
+      - appveyor AddMessage -Category Information "Downloading NewPascal %NewPascalVersion%..."
+      - appveyor DownloadFile https://github.com/newpascal/newpascal/releases/download/np-%NewPascalVersion%/newpascal.zip
```

Comment:
Why is this newpascal dependency with NodeJS and other stuff here, which should not be necessary for building USDX?

Reply:
Uh, nothing changed here. The install, build_script, after_build, and artifacts sections for the Windows build have just been reindented. Look at the changes with `git diff -w`.
And where do you see NodeJS?

Reply:
see NodeJS in https://github.com/newpascal/newpascal/blob/master/appveyor.yml

Reply:
Yeah, but those are the build rules for newpascal. We don't build newpascal. We just install the prebuilt binaries.
And the newpascal version is still the same old v1.0.50 we have been using since September 2018.